### PR TITLE
BaSP-MR-8: Classes Get, Get by ID & Create

### PR DESCRIPTION
**Pull request by keiler-art**
Created schema, validations and controllers (get, get by id and create) for classes. Also added the corresponding routes.

### CODE REVIEW

File: src/controllers/classes.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      });
+    }
+    return res.status(200).json({
+      message: 'Class found',
+      data: foundClass,
+      error: false,
+    });
+  } catch (error) {
+    return res.status(500).json({
+      message: 'Server error',
```
Comment (AleMac17) [inline src/controllers/classes.js:46]: In the message of the catch return, we need to put an error.
for example "message: error,"

File: src/controllers/classes.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import Class from '../models/Class';
+
+const getClasses = async (req, res) => {
+  try {
+    const classes = await Class.find({});
```
Comment (AleMac17) [inline src/controllers/classes.js:5]: Inside the find method, we only use (); without braces.
for example : "Class.find();"

**Review by AleMac17 — CHANGES_REQUESTED** [review]

File: src/controllers/classes.js (hunk 2 above)
Comment (keiler-art) [reply src/controllers/classes.js]: Done!

File: src/controllers/classes.js (hunk 1 above)
Comment (keiler-art) [reply src/controllers/classes.js]: Already done.

**Review by LucianoAlarcon — APPROVED** [review]

**Review by nicolasblasco — APPROVED** [review]

**Pull request merged**